h/buildext.py

```diff
@@ -179,10 +180,12 @@ def build_chrome(args):
     copytree('h/browser/chrome/content', 'build/chrome/content')
     copytree('h/browser/chrome/help', 'build/chrome/help')
     copytree('h/browser/chrome/images', 'build/chrome/images')
-    copytree('h/browser/chrome/lib', 'build/chrome/lib')
     copytree('h/static/images', 'build/chrome/public/images')
-    shutil.copyfile(
-        'h/static/scripts/blocklist.js', 'build/chrome/lib/blocklist.js')
+
+    os.makedirs('build/chrome/lib')
```

Comment:
Hmm, I still have a slight preference for putting the bundles in their own dir separate from the manifest and separate from any vendor code which is not bundled (eg. PDF.js)